File: WebApp/Models/BC/BC_WHCHKID.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class BC_WHCHKID : JCLib.Mvc.BaseModel
    {
        public string WH_NO { get; set; }
        public string WH_CHKUID { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }


        public string WH_CHKUID_NAME { get; set; }
        public string WH_NAME { get; set; }
        public string WH_GRADE { get; set; }
        public string WH_KIND { get; set; }

        public string ITEM_STRING { get; set; }


        public string IS_SELECTED { get; set; }
        public string HAS_ENTRY { get; set; }
    }
}

File: WebApp/Scripts/app/model/BC_WHCHKID.js
Ext.define('WEBAPP.model.BC_WHCHKID', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'WH_NO', type: 'string' },
        { name: 'WH_CHKUID', type: 'string' },
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' },

        { name: 'ITEM_STRING', type: 'string' },

        { name: 'WH_CHKUID_NAME', type: 'string' },
        { name: 'WH_KIND', type: 'string' },
        { name: 'WH_GRADE', type: 'string' },
        { name: 'WH_NAME', type: 'string' },
    ]
});